src/lib.rs

```diff
@@ -1,777 +1,13 @@
-//! This crate contains the types and functions for interacting with the Valhalla API.
-//!
-//! At the moment, only the routing API is implemented.
-//!
-//! # Examples
-//!
-//! ```
-//! use valhalla_client::{Valhalla, Location, Manifest};;
-//! let valhalla = Valhalla::default();
-//!
-//! let manifest = Manifest {
-//!     locations: vec![Location::new(4.9041, 52.3676), Location::new(5.1214, 52.0907)],
-//!     costing: valhalla_client::Costing::Bicycle,
-//!     ..Default::default()
-//! };
-//!
-//! let response = valhalla.route(manifest).unwrap();
-//!
-//! println!("{:#?}", response);
-//!
-//! // If the gpx feature is enabled, you can convert the response to a gpx::Gpx object
-//! // let gpx = response.trip.into();
-//! ```
-// Documentation: https://valhalla.github.io/valhalla/api/
+#![doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/README.md"))]
```

Comment:
In case you are concearned: yes, I obv. tested this ^^
![image](https://github.com/user-attachments/assets/9ac017b4-4c51-40e2-af0c-bbe01cc16714)